lib/aca_entities/medicaid/ios/functions/ssp_member__c_builder.rb

```diff
@@ -0,0 +1,206 @@
+# frozen_string_literal: true
+
+module AcaEntities
+  module Medicaid
+    module Ios
+      module Functions
+        # build SSP_Member__c for IOS transform
+        class SspMemberCBuilder
+          include ::AcaEntities::Operations::Transforms::HashFunctions
+
+          GENDER_MAP = {  'female' => 'F', 'male' => 'M' }.freeze
```

Comment:
Having all these hashes here is fine, but if we thought we would be referencing them across different transforms/operations, we could also create a types file.

Reply:
Moved to a newly created types file